### DISCUSSION TITLE
ResolutionContext has no public constructor

### DISCUSSION THREAD

**Discussion by gauravpatni123** [LuckyPennySoftware/AutoMapper#4134] [NA]
I am unable to create instance of **ResolutionContext** as it is internal in updated version (I am in version 12.0) , so my question is how I can mock it like before (previously I was in version 8.0 ) , please see the below code snippet , any help on this will be appreciated .
Below is the code : - 

 public class JobServiceClientOfValueResolverTests
    {
  private ResolutionContext _resolutionContext;
        private string _result;
        private Resource _resource2;

        public JobServiceClientOfValueResolverTests() {
            _destination = new JobViewDto();
            //_resolutionContext = new ResolutionContext(
            //    new Mock<IMappingOperationOptions>().Object,
            //    new Mock<IRuntimeMapper>().Object
            //);
        }

        private void When_Mapped()
        {
            _result = new JobServiceClientOfValueResolver().Resolve(_source, _destination, _destination.ClientOf, _resolutionContext);
        }
}

**Comment by lbargaoanu** [comment]
https://docs.automapper.org/en/latest/10.0-Upgrade-Guide.html#the-resolutioncontext-no-longer-has-a-public-constructor
